ui-tests/page-objects/meeting-page.js

```diff
@@ -0,0 +1,39 @@
+const { URL } = require('url');
```

Comment:
May be a stupid question: don't we use flow for these files? Or is it too early for that?

Reply:
I'd say not using flow has more to do with my own inabilities coupled with the need to quickly blast through prototyping. I'm assuming a lot of the existing code will change as a more refined version emerges, and afterwards I'd want to add flow for consistency with jitsi-meet. I didn't even really want jsdocs right now but thought they'd be helpful in case someone else that's not a js person was going to make changes.